modules/util/LoggerInit.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Logger initialization module
+ * This module sets up the enhanced logging transport and must be imported
+ * before any other modules that create loggers.
+ */
+
+// Import LoggerTransport which will automatically replace the console transport
```

Comment:
Not a fan of doing this at import time. it should likely be a function imported in JitsiMeetJS which we can call instead, early in JM?